### ISSUE TITLE
Netlify preset 'netlify-edge' Error

### ISSUE THREAD

**Issue by markus-gx** [nuxt/framework#7592]
### Environment

Netlify 

### Reproduction

Just create a Nuxt 3 Project and deploy it on netlify with the following preset: 

```
nitro: {
  preset: 'netlify-edge'
}
```

### Describe the bug

Netlify support suggested to raise a bug ticket here: 

> 
>  believe this is a problem with Nuxt and the way they're generating an Edge Function. It could be worth trying to post an issue here: https://github.com/nuxt/framework/issues.
>  
> The reason why I believe it is because, Nuxt is responsible for generating an Edge Function and there seems to be something wrong with that is being generated. But again, if they have any other ideas, you can let us know.

### Logs

```shell
────────────────────────────────────────────────────────────────
3:03:21 PM:   2. Edge Functions bundling                                    
3:03:21 PM: ────────────────────────────────────────────────────────────────
3:03:21 PM: ​
3:03:21 PM: Packaging Edge Functions from .netlify/edge-functions directory:
3:03:21 PM:  - server
3:03:24 PM: ​
3:03:24 PM: ────────────────────────────────────────────────────────────────
3:03:24 PM:   Internal error during "Edge Functions bundling"               
3:03:24 PM: ────────────────────────────────────────────────────────────────
3:03:24 PM: ​
3:03:24 PM:   Error message
3:03:24 PM:   Error: [
3:03:24 PM:     {
3:03:24 PM:       "instancePath": "/routes/0/pattern",
3:03:24 PM:       "schemaPath": "#/properties/routes/items/properties/pattern/errorMessage",
3:03:24 PM:       "keyword": "errorMessage",
3:03:24 PM:       "params": {
3:03:24 PM:         "errors": [
3:03:24 PM:           {
3:03:24 PM:             "instancePath": "/routes/0/pattern",
3:03:24 PM:             "schemaPath": "#/properties/routes/items/properties/pattern/format",
3:03:24 PM:             "keyword": "format",
3:03:24 PM:             "params": {
3:03:24 PM:               "format": "regexPattern"
3:03:24 PM:             },
3:03:24 PM:             "message": "must match format \"regexPattern\"",
3:03:24 PM:             "emUsed": true
3:03:24 PM:           }
3:03:24 PM:         ]
3:03:24 PM:       },
3:03:24 PM:       "message": "must match format /^\\^.*\\$$/"
3:03:24 PM:     }
3:03:24 PM:   ]
```

**Comment by markus-gx** [comment]
@danielroe Thanks for the fix/merge. Tried it and the deployment runs as expected. Sadly Netlify throws an error when trying to access the site. Maybe you can reproduce this or something?

```
8:48:50 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
8:48:50 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
8:50:04 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
8:50:16 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
8:50:16 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
8:50:25 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
8:50:25 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
8:50:39 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
8:50:39 AM: TypeError: Cannot read properties of undefined (reading '__EXTEND_POINT__')
    at file:///root/.netlify/edge-functions/server.js:1:509366
```

**Comment by danielroe** [comment]
@markus-gx I'm not getting that in a new Nuxt project. Please do create a new issue with a reproduction and I'll happily look into it 🙏

**Comment by danielroe** [comment]
This should be resolved in nitropack v0.5.4. which you can get with `yarn upgrade nuxt` or `npm upgrade nuxt`.

Let me know if not and I'll reopen.
